docs/using-the-probo-app/projects.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: "docs"
+title: Probo Projects
+class: documentation
+permalink: /app/projects/
+published: true
+---
+A Probo Project is a repository or group of repositories inside of a Probo Organization. The Projects overview tab in a Probo Organization is where all Probo Projects for that organization are listed with information specific to that project's resources used and latest Probo Build. Probo Projects are the repositories that have been synced and enabled from GitHub and Bitbucket. The repositories are labeled GitHub with the Octocat logo and Bitbucket with the Bucket logo to easily distinguish between a GitHub or Bitbucket project repository.
```

Comment:
> A Probo Project is a repository **or group of repositories** inside of a Probo Organization.

Is this true? I thought a Probo Project == a single activated repository. What does a group of repositories as a project look like?

Reply:
This was something that Dan described to me that I believe is planned. Perhaps I should reword it for now.

Reply:
If it's not a feature in production, it shouldn't be in the docs. We can always have PRs on deck for new features. Otherwise we will confuse our users.

Reply:
Agreed. I was getting ahead of myself there.

Reply:
Updated.